src/ServicePulse.Host/app/js/views/redirect/view.html

```diff
@@ -48,8 +48,8 @@
                         <div isolate-click class="col-sm-12">
                             <p class="small">
 
-                                <button type="button" class="btn btn-link btn-sm" confirm-title="Confirm action" confirm-click="vm.deleteRedirect(redirect, 'Redirect was deleted', 'Failed to delete redirect')" 
-                                        confirm-message="Are you sure you want to end the redirect?">
+                                <button type="button" class="btn btn-link btn-sm" confirm-title="Are you sure you want to end the redirect?" confirm-click="vm.deleteRedirect(redirect, 'Redirect was deleted', 'Failed to delete redirect')" 
+                                        confirm-message="Without this redirect retried messages will be send to its destination queue.">
```

Comment:
`Once the redirect is ended, any affected messages will be sent to the original destination queue. Ensure this queue is ready to accept messages again.`